tests/EndpointParameterMiddlewareTest.php

```diff
@@ -90,7 +90,7 @@ public function testCorrectlyOutputsHost(
         });
 
         $handler = $list->resolve();
-        $handler($command, new Request('POST', $endpoint));
+        $handler($command, new Request('POST', $endpoint))->wait(false);
```

Comment:
I found out why this was probably added.  

When running this test using PHP 8.0 and PHPUnit 9.x, this test is marked as a risky test because it does not run any assertions.  When adding `->wait(false)`, this disappears and the assertions are actually being done.